runtime-sdk/src/types/transaction.rs

```diff
@@ -148,6 +148,9 @@ pub enum CallResult {
 
         #[serde(rename = "code")]
```

Comment:
why are these renamed?

Reply:
To make it explicit how the fields are called when serialized.

Reply:
hm ok